variegated-hal/src/lib.rs

```diff
@@ -195,7 +193,6 @@ pub trait BoilerFillMechanism {
     fn get_filling_state(&self) -> Result<bool, BoilerFillMechanismError>;
 }
 
-#[async_trait]
```

Comment:
This doesn't work. The `async_trait` crate is still needed, because the traits are otherwise not dyn-compatible.